README.md

```diff
@@ -60,21 +60,27 @@ The API is based on the [FastAPI](https://fastapi.tiangolo.com/) library.
 
 ## Project Set Up
 * In your work directory,
-  * Run the script ./init.sh to create the image and set up all the environment variables used in the application
+  * Run the script `./init.sh` to create the image and set up all the environment variables used in the application
   * Start the container with the service involved in the search
   `docker-compose up -d`
+  **Note**: The container will take some minutes to start because it will start:
+    * the Solr server with authentication. In docker, all the users (`admin`, `solr`, `fulltext`) use the same password `solrRocks`
+    * the application to search the documents in the Solr server
+    * the API to search the documents in the Solr server
+    * a service to index some data in the Solr server to test the application
 
 
 ### Prerequisites
 * Docker
 * Code Editor
-* Python 3 and Poetry (If you want to run the application in your local environment)
+* Python 3 and Poetry (If you want to run the application in your local environment). See the installation section below.
   * To access to prod Solr server, you need it, 
     * to have a VPN connection to the HathiTrust network
     * to set up an ssh tunnel `ssh -L8081:localhost:8081 squishee-1.umdl.umich.edu`
     * to run the application in your local environment with the parameter `--env prod`.We don't have an 
     acceptable alternative, nor is it necessary to set up access to the production server via a Docker file.
-  * To query production, you will have to run the application locally and open and ssh connection to squishee-1. 
+  * To query production, you will have to run the application locally and open and ssh connection to squishee-1.
```

Comment:
squishee-1 will be retired, so we should list the new one (`macc-ht-solr-lss-1`) or reference some external documentation in confluence